hash_practice/exercises.py

```diff
@@ -2,18 +2,45 @@
 def grouped_anagrams(strings):
     """ This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n)
+        Space Complexity: O(1)

```

Comment:
As a key-value pair gets added to `anagram_dict` for every word in the list, this algorithm is not O(1) in space complexity, but O(n) where `n` is the number of strings in the list.